src/is-supported.ts

```diff
@@ -5,7 +5,7 @@ export function isSupported (): boolean {
   if (!mediaSource) {
     return false;
   }
-  const sourceBuffer = SourceBuffer || (window as any).WebKitSourceBuffer;
+  const sourceBuffer = (self as any).SourceBuffer || (self as any).WebKitSourceBuffer as SourceBuffer;
```

Comment:
```suggestion
  const sourceBuffer = self.SourceBuffer || (self as any).WebKitSourceBuffer as SourceBuffer;
```

`self.SourceBuffer` already has the correct type

Reply:
Maybe we need to update TypeScript?
```
$ npm run type-check

> tsc --noEmit

src/is-supported.ts:8:29 - error TS2339: Property 'SourceBuffer' does not exist on type 'Window'.

8   const sourceBuffer = self.SourceBuffer || (self as any).WebKitSourceBuffer as SourceBuffer;
```

Reply:
Oh maybe. I just checked locally in vscode and it had the types. If it doesn’t cause any issues I think keeping typescript up to date makes sense